include/nil/blueprint/parser.hpp

```diff
@@ -110,15 +110,16 @@ namespace nil {
         template<typename BlueprintFieldType, typename ArithmetizationParams>
         struct parser {
 
-            parser(long stack_size, boost::log::trivial::severity_level log_level, std::uint32_t max_num_provers, std::uint32_t target_prover_idx, const std::string &kind = "",
-                    print_format output_print_format = no_print, bool check_validity = false) :
+            parser(long stack_size, boost::log::trivial::severity_level log_level, std::uint32_t max_num_provers, std::uint32_t target_prover_idx, generate_flag gen_flag,
+                   const std::string &kind = "", print_format output_print_format = no_print, bool check_validity = false) :
```

Comment:
is kind needed?

Reply:
yes, it's define kind of policy. By the way it was not changed in scope of this PR